scripts/feature_tests/README.md

```diff
@@ -117,14 +126,6 @@ export CONTAINER_RUNTIME=podman
 export NUM_NODES=4
 ```
 
-Environment variables needed for upgrade tests running clusterctl,
-set as default values:
-
-```bash
-export CAPM3RELEASE=v0.3.2
```

Comment:
By the way, why are removing these variables?

Reply:
Please see above explanation.